docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,245 @@
+---
+id: googledrive
+title: Google Drive Connector
+description: Create empty folders or documents from templates from your BPMN process.
+---
+
+The **Google Drive Connector** allows you to create empty folders or documents on [Google Drive](https://drive.google.com/) from templates from your BPMN process.
+
+## Prerequisites
+
+To start working with the **Google Drive Connector**, a relevant OAuth token must be configured and stored as a Secret in the user's cluster. The token must have permission to read and create a folder and/or files from a desired Google Drive instance. Follow the steps from the [appendix](#appendix--faq) to find out more about creating an OAuth token and giving relevant permissions.
+
+## Create a Google Drive Connector task
+
+Currently, the **Google Drive Connector** supports two types of templates: create a folder and create a file from a template.
```

Comment:
Pls update text to:
"Currently, the **Google Drive Connector** supports two types of operations: create a folder and create a file from a template."

Reply:
Addressed in a new revision. Will publish shortly.